### ISSUE TITLE
Add CITATION file

### ISSUE THREAD

**Issue by willynilly** [lmfit/uncertainties#213]
Github support CITATION.cff files.  I was thinking it would be good to add one:  https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-citation-files

**Comment by wshanks** [comment]
Note how the original author asked for the package to be cited. I am not sure if or how this should ever change since he has handed over maintenance to the lmfit team:

https://github.com/lmfit/uncertainties/blob/e9b82f045f423452cfaeddc0b776905c05187e28/doc/index.rst?plain=1#L385-L392

**Comment by newville** [comment]
Adding a Zenodo citation for this project would be nice.  Certainly, @lebigot  E. O. Lebigot would be the lead author of that.  I would suggest even having E. O. Lebigot as the sole author -- the commit history is pretty clear.

**Comment by lebigot** [comment]
Honestly, I'm not against having all the contributors listed, but I'm sensitive to the commiters arguments: a single author would represent the commit history (and the creation effort) more clearly indeed. Thanks!

**Comment by wshanks** [comment]
Indeed, I see no reason to change the citation at this time. What I was not sure about was how we would agree on changing it in the future if there are further contributions, but that is a question for a later date :slightly_smiling_face:

**Comment by newville** [comment]
@wshanks @lebigot I enabled the Zenodo webhook so that, on the next release, a DOI will be created that can be used as the citation.   Still slowly working on the docs, and hope to have more time for this next week.

**Comment by lebigot** [comment]
Thanks!!
